src/components/shareable-url-modal/shareable-url-modal.scss

```diff
@@ -91,29 +103,49 @@
     }
   }
 
+  &__content-description-title:nth-of-type(2)::before {
```

Comment:
Which element we are trying to select here? Is it possible to avoid nth-of-type and use targeted class instead ?

Reply:
yup good point, let me do that 👍